snap/snapcraft.yaml

```
@@ -1,7 +1,27 @@
 name: authd-msentraid
-summary: MS EntraID Broker for authd
+summary: Authd - MS Entra ID Cloud Authentication for Ubuntu
 description: |
-  Broker that enables OIDC authentication with MS EntraID group management for authd.
+  Authd is a powerful authentication service for Ubuntu, designed to integrate with cloud identity providers like
+  Microsoft Entra ID. It delivers a secure, flexible solution for organizations and individuals who are transitioning
+  to cloud-based identity management on Ubuntu workstations and servers.
+  Authd uses the Oauth Device Authorization Grant and ensures a consistent and secure login experience across
```

Same as https://github.com/ubuntu/authd-oidc-brokers/pull/437#discussion_r1995718639

```suggestion
  Authd uses the OAuth Device Authorization Grant and ensures a consistent and secure login experience across
```